doc/whats_new.rst

```diff
@@ -19,6 +19,8 @@ Current
 Changelog
 ~~~~~~~~~
 
+- :func:`mne.BaseRaw.set_annotations` now synchronizes ``orig_time`` with ``meas_date`` before storing within the raw object by `Joan Massich`_
```

Comment:
This doesn't really say what this change implies to the user. Is it that this relationship used to hold in `master`:
```
annot = ...
raw.set_annotations(annot)
assert np.array_equal(annot.onset, raw.annotations.onset)
```
But now it does not? If so, something like "after setting annotations, raw.annotations.orig_time will be changed to match raw.info['meas_date']` and `raw.annotations.onset` will be shifted accordingly" or so. And then this description needs to go under the BUG section because it's a backward-incompatible change with no deprecation cycle.

But if this is not what "synchronizes" implies, it should say what it actually means in terms of how public attributes accessible to the user have changed.

Reply:
It's exactly that. So it would be a bug 'cos is back- incompatible, but it does not really look like a bug.

Shall we do a depreciation cycle